examples/multisig_transfer.ts

```diff
@@ -93,11 +93,39 @@ function createOrApproveMultisigProposal<
       other_signatories: signatories.filter((value) => value !== pair.publicKey),
       store_call: false,
       max_weight: {
-        ref_time: 500_000_000n,
+        ref_time: 0n,
         proof_size: 0n,
       },
       maybe_timepoint: maybeTimepoint as Rest[0],
     }),
+  })
+    .feeEstimate
+    .access("weight")
+    .next((weight) => {
+      return {
+        ref_time: BigInt(weight.ref_time),
+        proof_size: BigInt(weight.proof_size),
+      }
+    })
+  return extrinsic({
+    sender: C.compat.multiAddressFromKeypair(pair),
+    palletName: "Multisig",
+    methodName: "as_multi",
+    args: C.Z.rec({
+      threshold: THRESHOLD,
+      call: {
+        type: "Balances",
+        value: {
+          type: "transfer_keep_alive",
+          dest: C.compat.multiAddressFromKeypair(T.dave),
+          value: 1_230_000_000_000n,
+        },
+      },
+      other_signatories: signatories.filter((value) => value !== pair.publicKey),
+      store_call: false,
+      max_weight: maxWeight,
+      maybe_timepoint: maybeTimepoint as Rest[0],
```

Comment:
@harrysolovay the extrinsic data is almost the same but the spread operator doesn't work well with `C.Z.rec`

Would you know a better alternative?

I could factor out the `args` within `Z.rec`, WDYT?

Reply:
Or, given that it's an example, code duplication might be fine

Reply:
I think you want to calculate the weight for the `Balances.transfer_keep_alive` call, not the `Multisig.as_multi`, which would solve this duplication.

Reply:
Yea, I believe you'll be able to get the appropriate weight using the encoded inner extrinsic.

On a more general note though...

- A bit of redundancy in examples can definitely help with reader comprehension
- In this particular case, I suppose we could extract the common fields –– another way we'll make this example nicer: @tjjfvi is working on #368, which will enable us to generate/use narrow call data types (and their factory fns)

Reply:
Yeah, I'll resolve the duplication in the codegen branch using the call factories once this is merged – no need to address it now.